modules/distribution/carbon-home/conf/dashboard/deployment.yaml

```diff
@@ -365,6 +365,22 @@ wso2.datasources:
         connectionTestQuery: SELECT 1
         validationTimeout: 30000
         isAutoCommit: false
+        
+  - name: IS_ANALYTICS_DB
+    description: "The datasource used for WSO2 Identity Server Analytics dashboard feature"
```

Comment:
"The data source used for WSO2 IS Analytics statistics"

Reply:
Done.